log/sequencer.go

```diff
@@ -188,11 +190,23 @@ func (s Sequencer) sequenceLeaves(mt *merkle.CompactMerkleTree, leaves []*trilli
 			return nil, nil, err
 		}
 		// The leaf has now been sequenced.
-		leaves[i].LeafIndex = seq
-		leaves[i].IntegrateTimestamp, err = ptypes.TimestampProto(s.timeSource.Now())
+		leaf.LeafIndex = seq
+		integrateTS := s.timeSource.Now()
+		leafIntegrateTimestamp, err = ptypes.TimestampProto(integrateTS)
```

Comment:
This is not being assigned to the leaf anymore.

Reply:
typo, fixed